plugins/modules/packaging/language/pipx_info.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2021, Alexei Znamensky <russoz@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: pipx_info
+short_description: Rretrieves information about applications installed with pipx
+version_added: 5.6.0
+description:
+    - Retrieve details about Python applications installed in isolated virtualenvs using pipx.
+options:
+    name:
+        description:
+            - Name of an application with C(pipx).
+        type: str
+    include_deps:
+        description:
+            - Include dependent packages in the output.
+        type: bool
+        default: false
+    include_injected:
+        description:
+            - Include injected packages in the output.
+        type: bool
+        default: false
+    include_raw:
+        description:
+            - Returns the raw output of C(pipx list --json).
+            - The raw output is not affected by I(include_deps) or I(include_injected).
+        type: bool
+        default: false
+    executable:
+        description:
+            - Path to the C(pipx) installed in the system.
+            - >
+              If not specified, the module will use C(python -m pipx) to run the tool,
+              using the same Python interpreter as ansible itself.
+        type: path
+notes:
+    - This module does not install the C(pipx) python package, however that can be easily done with the module M(ansible.builtin.pip).
+    - This module does not require C(pipx) to be in the shell C(PATH), but it must be loadable by Python as a module.
```

Comment:
Out of curiosity: why is that required?

Reply:
Because we invoke `pipx` as `<python> -m pipx`, allowing `pipx` installed both at system-wide level and user-wide level. In fact, in the integration test we install `pipx` with:
```yaml
- name: install pipx
  pip:
    name: pipx
    extra_args: --user
```

Reply:
Because we invoke `pipx` as `<python> -m pipx`, allowing `pipx` installed both at system-wide level and user-wide level. In fact, in the integration test we install `pipx` with:
```yaml
- name: install pipx
  pip:
    name: pipx
    extra_args: --user
```